packages/vite-plugin/jest.config.ts

```diff
@@ -1,7 +1,10 @@
 import { type Config } from "jest";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const packageJson = require("./package.json") as { version: string };
+const packageJson = require("./package.json") as {
```

Comment:
can we use `await import` here instead of the require?

Reply:
Will do this in my follow up PR that migrates from Jest to Vitest.